content/hi/includes/task-tutorial-prereqs.md

```diff
@@ -1,8 +1,8 @@
 आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को 
-समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सकें। हमारी सलाह है की इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करे जो कि कंट्रोल प्लेन 
+समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। हमारी सलाह है कि इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करें जो कि कंट्रोल प्लेन 
 होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप 
```

Comment:
This one is more appropriate

समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। हमारी सलाह है कि इस ट्यूटोरियल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करें जो कि कंट्रोल प्लेन
होस्ट की तरह कार्य ना करे । अगर आपके पास पहले से क्लस्टर नही है, तो आप